test/jtreg-ext/requires/VMProps.java

```diff
@@ -123,6 +123,10 @@ public Map<String, String> call() {
         map.put("vm.continuations", this::vmContinuations);
         // vm.graal.enabled is true if Graal is used as JIT
         map.put("vm.graal.enabled", this::isGraalEnabled);
+        // vm.libgraal is true if the libgraal shared library file is present
+        map.put("vm.libgraal", this::hasLibraal);
```

Comment:
May be the property also should be named "vm.hasLibgraal" to be clear what it means.

Reply:
I'm not so sure. The current naming seems consistent with other existing properties:
```
map.put("vm.jvmti", this::vmHasJVMTI);
map.put("vm.jvmci", this::vmJvmci);
```
However, I can change it if you insist.

Reply:
Ok, `hasLibgraal` is fine.
In what sense is libgraal not part of the VM? I would have thought that a JIT compiler is more "vm" than "jdk"?

Reply:
In a sense that the libgraal **file** is separate from VM and its generation is not part of HotSpot VM build.
`hasLibgraal` property checks the presence of this separate file and not a feature which is part of VM build (like jvmci).

Reply:
"vm.libgraal.enabled" is VM's property because it enables VM's Graal JIT version with use code in libgraal.

Reply:
On other hand, it is not really big matter. I am fine with "vm. hasLibgraal" since it is used only by VM.

Reply:
You've convinced me - I'll rename it to `jdk.hasLibgraal`.